Dockerfile

```diff
@@ -1,8 +1,10 @@
 # VERSION   0.1
-FROM        debian:unstable
+FROM        debian:stable
```

Comment:
Shouldn't this explicitly be `jessie` for more stability, especially when we get closer to `stretch`?